datajoint/blob.py

```diff
@@ -251,6 +263,34 @@ def pack_recarray(self, array):
     def read_sparse_array(self):
         raise DataJointError('datajoint-python does not yet support sparse arrays. Issue (#590)')
 
+    def read_int(self):
+        return int(self.read_value('int64'))
+
+    @staticmethod
+    def pack_int(v):
+        return b"\x0a" + np.array(v, dtype='int64').tobytes()
+
+    def read_bool(self):
+        return bool(self.read_value('bool'))
+
+    @staticmethod
+    def pack_bool(v):
+        return b"\x0b" + np.array(v, dtype='bool').tobytes()
+
+    def read_complex(self):
+        return complex(self.read_value('complex128'))
+
+    @staticmethod
+    def pack_complex(v):
+        return b"\x0c" + np.array(v, dtype='complex128').tobytes()
```

Comment:
We could utilize `decimal` packing here for the same reasons as `float` below. Python seems to capture the first 53 bits for each the `real` part and the `complex` part.

Reply:
here Python is not doing anything special and just uses the standard IEEE 754 encoding.